src/sidebar/services/test/annotations-test.js

```diff
@@ -29,7 +29,7 @@ class FakeSearchClient extends EventEmitter {
   }
 }
 
-describe('annotations', () => {
+describe('sidebar/services/annotationService', () => {
```

Comment:
```suggestion
describe('sidebar/services/annotations', () => {
```

Describe block name should be the path (relative to `src/`) OR the export name, but not a mix of the two.

Reply:
Sorry I missed this one. Will update before I merge.